packages/terra-slide-panel-manager/src/terra-dev-site/test/slide-panel-manager/DisclosureComponent.jsx

```diff
@@ -57,7 +57,7 @@ class DisclosureComponent extends React.Component {
         dimensions,
         content: {
           key: `DemoContainer-${newIndex}`,
-          component: <WrappedDisclosureComponent identifier={`DemoContainer-${newIndex}`} nestedIndex={newIndex} />,
+          component: <WrappedDisclosureComponent identifier={`DemoContainer-${newIndex}`} nestedIndex={newIndex} disclosureType={disclosureType} />,
```

Comment:
if a modal manager was provided above the slide panel manager, subsequent disclosures were showing up as modals, not slide panels